src/core/Akka.Tests/Actor/Scheduler/BugFix7247Spec.cs

```diff
@@ -0,0 +1,67 @@
+// -----------------------------------------------------------------------
+//  <copyright file="7247BugFixSpec.cs" company="Akka.NET Project">
+//      Copyright (C) 2009-2024 Lightbend Inc. <http://www.lightbend.com>
+//      Copyright (C) 2013-2024 .NET Foundation <https://github.com/akkadotnet/akka.net>
+//  </copyright>
+// -----------------------------------------------------------------------
+
+using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Akka.Actor;
+using Akka.Routing;
+using Akka.TestKit;
+using Akka.Util.Internal;
+using FluentAssertions;
+using Xunit;
+
+namespace Akka.Tests.Actor.Scheduler;
+
+public class BugFix7247Spec : AkkaSpec {
+    public sealed class NoCellActorRef : MinimalActorRef
+    {
+        public NoCellActorRef(ActorPath path, TaskCompletionSource<object> firstMessage)
+        {
+            Path = path;
+            FirstMessage = firstMessage;
+        }
+
+        protected override void TellInternal(object message, IActorRef sender)
+        {
+            FirstMessage.TrySetResult(message);
+        }
+
+        public TaskCompletionSource<object> FirstMessage { get; }
+
+        public override ActorPath Path { get; }
+        public override IActorRefProvider Provider => throw new NotImplementedException();
+    }
+
+    [Fact(DisplayName = "Should not send ScheduledTellMsg envelopes to IActorRefs with no cell")]
+    public async Task ShouldNotSendScheduledTellMsgToNoCellActorRefs()
+    {
+        // arrange
+        var tcs = new TaskCompletionSource<object>(TaskCreationOptions.RunContinuationsAsynchronously);
+        var noCellRef = Sys.As<ExtendedActorSystem>().Provider.CreateFutureRef(tcs);
+        var router = Sys.ActorOf(Props.Empty.WithRouter(new BroadcastGroup()));
+        var routee = Routee.FromActorRef(noCellRef);
+        router.Tell(new AddRoutee(routee));
+
+        await AwaitAssertAsync(async () =>
+        {
+            var allRoutees = await router.Ask<Routees>(GetRoutees.Instance);
+            allRoutees.Members.Count().ShouldBeGreaterThan(0);
+        });
```

Comment:
You can't target a `/temp` actor via `ActorSelection`, so I had to go through some extra steps in order to get the `FutureActorRef` added to the `BroadcastGroup` router here.